## set 2, Q1

In [2]:
from scipy import stats as stats

In [3]:
#As the manager start after 10 min of drop off so he has only 50 min to complete his work.

In [4]:
#probablity of it will take more than 50 for servicing:
1-(stats.norm.cdf(50,45,8))

0.26598552904870054

## Set 2 Q2
The current age (in years) of 400 clerical employees at an insurance claims processing center is normally distributed with mean  = 38 and Standard deviation  =6. For each statement below, please specify True/False. If false, briefly explain why.

A.	More employees at the processing center are older than 44 than between 38 and 44./n

B.	A training program for employees under the age of 30 at the center would be expected to attract about 36 employees.


In [5]:
#Probability of older than 44:
1-(stats.norm.cdf(44,38,6))

0.15865525393145707

In [6]:
#probability of age bitween 38 and 44:
stats.norm.cdf(44,38,6)-stats.norm.cdf(38,38,6)

0.3413447460685429

In [7]:
#No. of employee whose age is less than 30:
round(400*(stats.norm.cdf(30,38,6)))

36.0

## Set 2 Q3

###  If X1 ~ N(μ, σ2) and X2 ~ N(μ, σ2) are iid normal random variables, then what is the difference between 2 X1 and X1 + X2? Discuss both their distributions and parameters.   

### Answer:
#### X1 and X2 are random veriable in normal distribution so they are two diffrent value
####  as all value are diffrent X1 and X2 are diffrent hence the result of 2X1 and X1+X2 will be diffrent 

## Set 2 Q4
### Let X ~ N(100, 202). Find two values, a and b, symmetric about the mean, such that the probability of the random variable taking a value between them is 0.99. 

In [8]:
import scipy.stats as stats

In [9]:
#calulating z score of x2
z2=stats.norm.ppf(0.995)
print(z2)

2.5758293035489004


In [10]:
sd=20
mean=100

In [11]:
#calculating the x2 using z score formula
x2=(z2*sd)+mean
print(x2)

151.516586070978


In [12]:
#cross checking the probability
stats.norm.cdf(x2,100,20)

0.995

In [13]:
#calulating z score of x1
z1=stats.norm.ppf(0.005)
print(z1)

-2.575829303548901


In [14]:
#calculating the x2 using z score formula
x1=(z1*sd)+mean
print(x1)

48.483413929021985


In [15]:
#calulation of proability between x1 and x2
stats.norm.cdf(x2,mean,sd)-stats.norm.cdf(x1,mean,sd)

0.99

### Final answer opiton D. 48.5, 151.5 

## Set 2 Q5
### Consider a company that has two different divisions. The annual profits from the two divisions are independent and have distributions Profit1 ~ N(5, 32) and Profit2 ~ N(7, 42) respectively. Both the profits are in dollor Million. Answer the following questions about the total profit of the company in Rupees. Assume that $1 = Rs. 45


A.	Specify a Rupee range (centered on the mean) such that it contains 95% probability for the     annual profit of the company.

B.	Specify the 5th percentile of profit (in Rupees) for the company

C.	Which of the two divisions has a
larger probability of making a loss in a given year?

In [16]:
#CONVERSION IN RUPEES
# profit1~N(225,1440)
# profit1~N(315,1890)

In [17]:

p1_mean=225
p1_sd=1440
p2_mean=315
p2_sd=1890
'''
p1_mean=5
p1_sd=32
p2_mean=7
p2_sd=42

'''

'\np1_mean=5\np1_sd=32\np2_mean=7\np2_sd=42\n\n'

#### For company 1 Rupee range (centered on the mean) such that it contains 95% probability for the annual profit of the company.

In [18]:
#z score for 95% confidence interval
c1_z1 = stats.norm.ppf(0.975)
c1_z2= stats.norm.ppf(0.025)

In [19]:
#calulating rupee range for compnay 1
c1_higher_value=round((c1_z2*p1_sd)+p1_mean)
c1_lower_value=round((c1_z1*p1_sd)+p1_mean)
print('Rupee range that contain 95% of profit is ', c1_lower_value,  'to', c1_higher_value)


Rupee range that contain 95% of profit is  3047.0 to -2597.0


In [20]:
#cross checking result 
stats.norm.cdf(c1_higher_value,p1_mean,p1_sd)-stats.norm.cdf(c1_lower_value,p1_mean,p1_sd)

-0.9499717336727853

#### For company 2 Rupee range (centered on the mean) such that it contains 95% probability for the annual profit of the company.

In [21]:
#z score for 95% confidence interval
c2_z2 = stats.norm.ppf(0.975)
c2_z1 = stats.norm.ppf(0.025)

In [22]:
#calulating rupee range for compnay 2
c2_higher_value=round((c2_z2*p2_sd)+p2_mean)
c2_lower_value=round((c2_z1*p2_sd)+p2_mean)
print('Rupee range that contain 95% of profit is ', c2_lower_value,  'to', c2_higher_value)


Rupee range that contain 95% of profit is  -3389.0 to 4019.0


In [23]:
#cross checking result 
stats.norm.cdf(c2_higher_value,p2_mean,p2_sd)-stats.norm.cdf(c2_lower_value,p2_mean,p2_sd)

0.9499794676647622

#### 5th percentile of profit (in Rupees) for the company

In [24]:
#5th percentile of comapnay 1
#50th pecentile is mean = 225
c1_5th_percentile=p1_mean*0.1
print(c1_5th_percentile)

22.5


In [25]:
#5th percentile of comapnay 2
#50th pecentile is mean = 315
c2_5th_percentile=p2_mean*0.1
print(c2_5th_percentile)

31.5


### Set 3 Q 1

For each of the following statements, indicate whether it is True/False. If false, explain why.

    I.	The sample size of the survey should at least be a fixed percentage of the population size in order to produce representative results.
        Ans: True
        
    II.	The sampling frame is a list of every item that appears in a survey sample, including those that did not respond to questions.
        Ans: False-The sampling frame refers to a list of an item which responds to the question and not the ones which donot respond to the questions. 

    III.	Larger surveys convey a more accurate impression of the population than smaller surveys.
        Ans: True

## Set 3  Q 2

PC Magazine asked all of its readers to participate in a survey of their satisfaction with different brands of electronics. In the 2004 survey, which was included in an issue of the magazine that year, more than 9000 readers rated the products on a scale from 1 to 10. The magazine reported that the average rating assigned by 225 readers to a Kodak compact digital camera was 7.5. For this product, identify the following:

A.	The population  :  All readers

B.	The parameter of interest  : Rating scale on 1-10

C.	The sampling frame  :  Readers who read PC magzine

D.	The sample size : 225

E.	The sampling design : Survey of PC magzine

F.	Any potential sources of bias or other problems with the survey or sample:
        We don't know if from all users of camera masure users were reading the magzine or not.
        We don't know if from all readers who use camera have rated it or not.
        


### Set 3 Q 3

For each of the following statements, indicate whether it is True/False. If false, explain why.

I.	If the 95% confidence interval for the average purchase of customers at a department store is $50 to $110, then $100 is a plausible value for the population mean at this level of confidence.

        Ans: False. 100 is closer to higer value 110 so it is not plausible to say 100 is mean value

II.	If the 95% confidence interval for the number of moviegoers who purchase concessions is 30% to 45%, this means that fewer than half of all moviegoers purchase concessions.
        
        Ans: True

III.	The 95% Confidence-Interval for μ only applies if the sample data are nearly normally distributed.
        Ans: False this rule can still be applicatble for highly skewed data or data which is not normally distributed


### Set 3 Q 4

What are the chances that  ?

#### Ans: D)  1


### Set 3 Q 5

In January 2005, a company that monitors Internet traffic (WebSideStory) reported that its sampling revealed that the Mozilla Firefox browser launched in 2004 had grabbed a 4.6% share of the market.

I.	If the sample were based on 2,000 users, could Microsoft conclude that Mozilla has a less than 5% share of the market?
        Ans:
        
        
II.	WebSideStory claims that its sample includes all the daily Internet users. If that’s the case, then can Microsoft conclude that Mozilla has a less than 5% share of the market?
        Ans: Yes


##### s3q5-1

### Set 3 Q 6

A book publisher monitors the size of shipments of its textbooks to university bookstores. For a sample of texts used at various schools, the 95% confidence interval for the size of the shipment was 250 ± 45 books. Which, if any, of the following interpretations of this interval are correct?

A.	All shipments are between 205 and 295 books.

####       Ans: NO. 95% of shippment are between 205 to 295 books

B.	95% of shipments are between 205 and 295 books.

####        Ans: Ture

C.	The procedure that produced this interval generates ranges that hold the population mean for 95% of samples.

####       Ans: True

D.	If we get another sample, then we can be 95% sure that the mean of this second sample is between 205 and 295.

#### Ans: Yes 

E.	We can be 95% confident that the range 160 to 340 holds the population mean.
 
 #### Ans: Yes


### Set 3 Q 7

Which is shorter: a 95% z-interval or a 95% t-interval for μ if we know that σ =s?

A.	The z-interval is shorter

B.	The t-interval is shorter

C.	Both are equal

D.	We cannot say

#### c) both are equal


##### Questions 8 and 9 are based on the following: To prepare a report on the economy, analysts need to estimate the percentage of businesses that plan to hire additional employees in the next 60 days.



### Set 3 Q 8

How many randomly selected employers (minimum number) must we contact in order to guarantee a margin of error of no more than 4% (at 95% confidence)?

A.	600

B.	400

C.	550

D.	1000


In [26]:
#### margin of error=z*(sqrt(p^(1-p^)/n)))

In [27]:
#conversion 
#N=[(Z/MOE)^2]*P(1-P)

In [28]:
#N is =
((1.95/0.04)**2)*(.5*(1-0.5))

594.140625

#### Ans. a) 600

### Set 3 Q 9

Suppose we want the above margin of error to be based on a 98% confidence level. What sample size (minimum) must we now use?

A.	1000

B.	757

C.	848

D.	543


In [29]:
#N is =
((2.33/0.04)**2)*(.5*(1-0.5))

848.265625

#### Ans. c) 848

In [30]:
#------------------------------